src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java

```diff
@@ -299,7 +308,7 @@ public String toString() {
 
 		@Exported
 		public AbstractBuild<?,?> getBuild() {
-			return build;
+			return (AbstractBuild) Run.fromExternalizableId(buildID);

```

Comment:
I would add class check just in case the data is messed up.
CheckForNull would be also useful. How is it handled in the downstream code?